jstests/oidc/lib/oidc_fixture.js

```diff
@@ -179,6 +206,16 @@ export class OIDCFixture {
         this.admin_conn._setOIDCIdPAuthCallback(callback.toString());
     }
 
+    /**
+     * Set up the OIDC fixture with audit logging enabled.
+     *
+     * This function configures audit path and calls the setup() function.
+     */
+    setup_with_audit() {
+        this.audit_path = OIDCFixture.allocate_audit_path();
+        this.setup();
+    }
```

Comment:
Can we, instead of introducing a dedicated function, just add the `with_audit` booleean parameter to the `setup` function and make the parameter `false` by default?